pom.xml

```diff
@@ -313,20 +319,20 @@
 				<artifactId>maven-enforcer-plugin</artifactId>
 				<version>3.0.0</version>
 				<executions>
-					<execution>
-						<id>enforce-maven</id>
-						<goals>
-							<goal>enforce</goal>
-						</goals>
-						<configuration>
-							<rules>
-								<requireMavenVersion>
-									<version>3.3.9</version>
-								</requireMavenVersion>
-							</rules>
-						</configuration>
-					</execution>
-				</executions>
+        <execution>
```

Comment:
please reformat the file (uing IDE)
project-keeper is unfortunately not formatting correctly

Reply:
Done